### PR TITLE
fix bug, recursuve=>recursive

### PR DESCRIPTION
**Pull request by wangzhen11aaa**
Words spell wrong.

### CODE REVIEW

**Comment by wangzhen11aaa** [comment]
1. Recursive equals true is good for cos.
2.We can ignore some files with some other suffix.

**Comment by Slach** [comment]
@wangzhen11aaa thanks a lot for contributing

File: pkg/new_storage/general.go [hunk 1]
```diff
@@ -105,17 +105,14 @@ func (bd *BackupDestination) BackupList() ([]Backup, error) {
 			})
 			return nil
 		}
-		mf, err := bd.StatFile(path.Join(o.Name(), "metadata.json"))
+		metadataFilePath := ""
```
Comment (Slach) [inline pkg/new_storage/general.go:108]: Did you run `./tests/integration/run.sh` to check your changes for other types of remote storage?

Comment (Slach) [reply pkg/new_storage/general.go:108]: also, look like recursive walking will list lot of files which is not folder with backup name

**Review by Slach — CHANGES_REQUESTED** [review]
look like your changes will increase resource usage for other types of remote storage

**Comment by Slach** [comment]
@wangzhen11aaa integration tests failed
look details
https://github.com/AlexAkulov/clickhouse-backup/runs/4699579162

File: pkg/new_storage/general.go (hunk 1 above)
Comment (wangzhen11aaa) [reply pkg/new_storage/general.go:108]: > also, look like recursive walking will list lot of files which is not folder with backup name

I think that is ok. Without "recursive=true", we can not get some information about some backup.

**Comment by wangzhen11aaa** [comment]
> @wangzhen11aaa integration tests failed look details https://github.com/AlexAkulov/clickhouse-backup/runs/4699579162

ok

**Pull request closed without merging**